### PR TITLE
feat: add development controller

### PR DESCRIPTION
**Pull request by NamelsKing**

### CODE REVIEW

File: src/dev/dev.controller.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  @ContentType('json')
+  @Description('Put any function that u want to test out')
+  async test(@QueryParams('param') param: string): Promise<string> {
+    return param;
+  }
+
+  @Get('/ddb/seeds')
+  @ContentType('json')
+  @Description('Update seed files for DynamoDb')
+  async getAccount(): Promise<{ status: string }> {
```
Comment (axejintao) [inline src/dev/dev.controller.ts:24]: should we rename this one

File: src/dev/dev.controller.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+@UseBefore(DevelopmentFilter)
+@Hidden()
+export class DevelopmentController {
+  @Inject()
+  developmentService!: DevelopmentService;
+
+  @Get('/test')
+  @ContentType('json')
+  @Description('Put any function that u want to test out')
+  async test(@QueryParams('param') param: string): Promise<string> {
```
Comment (axejintao) [inline src/dev/dev.controller.ts:17]: remove this one or wants to keep it?

**Review by axejintao — COMMENTED** [review]
looks good, build failing

File: src/dev/dev.controller.ts (hunk 1 above)
Comment (NamelsKing) [reply src/dev/dev.controller.ts:24]: sure doing

File: src/dev/dev.controller.ts (hunk 2 above)
Comment (NamelsKing) [reply src/dev/dev.controller.ts:17]: ye, i wonnna keep this for custom launches

**Review by axejintao — APPROVED** [review]

**Pull request merged**